spring-boot-project/spring-boot-devtools/src/test/java/org/springframework/boot/devtools/livereload/LiveReloadServerTests.java

```diff
@@ -88,19 +89,14 @@ void servesLivereloadJs() throws Exception {
 	void triggerReload() throws Exception {
 		LiveReloadWebSocketHandler handler = connect();
 		this.server.triggerReload();
-		Thread.sleep(200);
-		this.server.stop();
```

Comment:
I think we need to keep this call to `stop()`.

Reply:
The test class already has an `@AfterEach` which calls `this.server.stop();`.
I assumed the original author had written those 2 test cases first and only after that decided to have a general `@AfterEach` method (and maybe forgot to take the individual `stop()`s out).

I made sure that the tests run fine without them.
Do you think `stop()` should be kept nevertheless?

Reply:
Sorry, I'd missed that when reviewing the diffs in isolation. I agree there's no need to keep the `stop()` calls. To keep this change clean, I think it makes sense to remove the `stop()` calls separately. No need for you to do anything though. We can take care of it prior to merging this.